.circleci/config.yml

```diff
@@ -352,9 +396,11 @@ workflows:
             - 'Nomodule Build'
             - 'Module Build'
       - 'End-to-End Tests':
+          name: '⛓️ End-to-End Tests'
           <<: *push_and_pr_builds
           requires:
             - 'Nomodule Build'
+            - '⛓️ All Unit Tests'
```

Comment:
Was it your intent to make the E2E tests wait for the Unit tests to reduce VM usage? Instead, WDYT about getting rid of this dependency, and playing with the overall parallelism number (see other comment)?

Reply:
Did you read the explanation above about this dependency and why I think it's okay?

btw, I'm suspecting that parallelized jobs _do not_ use an extra VM - the documentation doesn't say anything about it, but my hint is that each job has the `🗳 1 / 83 parallel jobs ℹ️` indicator, and multiple parallelized jobs in my workflow have been running at the same time, saying `🗳 16 / 83 parallel jobs ℹ️` - so it looks like they're not constrained by the same 80 VMs limit

Reply:
Now that I'm re-reading my own explanations, I realize that this change only makes sense if the assumption in my previous comment on this thread is wrong - which is not what the signs point to. Reverted the change.